app/controllers/api/comments_controller.rb

```diff
@@ -12,11 +13,18 @@ def create
 
   private
 
+  def update_and_set_user
+    @user = MobileUser.find_by(auth_token: @auth_token)
+    @user.update(name: params.fetch(:username))
```

Comment:
Nie jestem pewien jak to działa - czy przy takim kodzie SQL-ka pójdzie nawet jak username tak naprawdę się nie zmieni? Możesz to sprawdzić? Jeżeli SQL-ka leci zawsze, to nie ma takiej potrzeby - powinna iść tylko jeżeli username się zmienił.